hasura/Dockerfile

```diff
@@ -0,0 +1,9 @@
+# https://github.com/hasura/graphql-engine
+FROM hasura/graphql-engine:v2.20.0-ce.cli-migrations-v3
+
+ENV HASURA_GRAPHQL_ENABLE_TELEMETRY=${HASURA_GRAPHQL_ENABLE_TELEMETRY:-false}
+ENV HASURA_GRAPHQL_ENABLED_LOG_TYPES=${HASURA_GRAPHQL_ENABLED_LOG_TYPES:-"startup, http-log, webhook-log, websocket-log, query-log"}
+ENV HASURA_GRAPHQL_EXPERIMENTAL_FEATURES=${HASURA_GRAPHQL_EXPERIMENTAL_FEATURES:-naming_convention}
+ENV HASURA_GRAPHQL_DEFAULT_NAMING_CONVENTION=${HASURA_GRAPHQL_DEFAULT_NAMING_CONVENTION:-graphql-default}
+
+COPY ./metadata /hasura-metadata
```

Comment:
since we're copying in the metadata at build time lets also set the `HASURA_GRAPHQL_METADATA_DIR` environment variable to this path, just to avoid potential issues if they change the default in the future.

Reply:
good idea:  [81ad5b1](https://github.com/sillsdev/languageforge-lexbox/commit/81ad5b1880fe69e06d65e01ef781afb22bd2f56f)